sass/civicrm-override/_calender.scss

```diff
@@ -0,0 +1,194 @@
+.ui-datepicker {
+  background: $crm-white;
+  border: 1px solid $datepicker-border-color;
+  border-radius: 0;
+  box-shadow: $box-shadow;
+  min-width: 160px;
+  padding: 10px;
+  width: auto;
+
+  &,
+  * {
+    border-radius: 0;
+    font-family: $font-family-base;
+  }
+
+  * {
+    background: none;
+    box-shadow: none;
+  }
+
+  &,
+  .ui-datepicker-calendar {
+    td[data-handler] a {
+      background: $gray-lighter;
+    }
+
+    td[data-handler] a.ui-state-highlight {
+      border-color: $brand-primary;
+    }
+  }
+}
+
+.ui-datepicker .ui-datepicker-header {
+  border: 0;
+  margin-bottom: 5px;
+
+  .ui-datepicker-next,
+  .ui-datepicker-prev {
+    box-shadow: none;
+    height: 28px;
+    top: 4px;
+
+    &::before {
+      display: inline-block;
+      line-height: 30px;
+    }
+  }
+
+  .ui-datepicker-prev::before {
+    @include fa-icon(13px, '\f0d9');
+    padding-left: 5px;
+  }
+
+  .ui-datepicker-next::before {
+    @include fa-icon(13px, '\f0da');
+    padding-left: 15px;
+  }
+
+  select {
+    appearance: none;
+    background: url('../svgs/drop-down-arrow.svg') no-repeat right;
+    display: inline-block;
+    font-family: $font-family-base;
+    font-size: 0.8em;
+    padding-left: 5px;
+    padding-right: 10px;
+    position: relative;
+    top: 1px;
+    width: 75px;
+  }
+}
+
+.ui-datepicker-header {
+  .ui-datepicker-next,
+  .ui-datepicker-prev {
+    border-radius: 2px;
+    cursor: pointer;
+    text-align: center;
+    width: 35px;
+
+    span {
+      background: none;
+    }
+
+    &:before {
+      color: $datepicker-gray-dark;
+      font-size: $font-size-medium;
+      font-weight: normal;
+      padding: 0;
+    }
+
+    &-hover {
+      background: $gray-lighter;
+      border: 0;
+      border-color: $datepicker-gray-darker;
+      color: $datepicker-gray-dark;
+    }
+  }
+
+  .ui-datepicker-prev {
+    &:before {
+      content: '\f053';
+    }
+
+    &-hover {
+      left: 2px;
+    }
+  }
+
+  .ui-datepicker-next {
+    &:before {
+      content: '\f054';
```

Comment:
Yea same comment i only replaced this with mixin instead of using content 
https://github.com/compucorp/uk.co.vedaconsulting.mosaico/pull/13/commits/14d325f0f2b8aa34418a6421b4991e4c1931437d